nestor_api/lib/git.py

```diff
@@ -72,9 +72,11 @@ def push(repository_dir: str, branch_name: str = "HEAD") -> None:
     io.execute(f"git push origin {branch_name} --tags --follow-tags", repository_dir)
 
 
-def rebase(repository_dir: str, branch_name: str) -> None:
+def rebase(repository_dir: str, branch_name: str, *, onto: str = None) -> None:
```

Comment:
Cool feature,  didn't know this was how it was enforced 👍

Reply:
@Kenny407 Yep, you can dig the subject by reading [this section](https://docs.python.org/3/tutorial/controlflow.html#more-on-defining-functions) of the Python tutorial which explains well all of the possible function arguments 🙂